_posts/2017-03-05-artsy-technology-stack-2017.md

```diff
@@ -49,11 +49,15 @@ We have three major CMS projects:
 [Writer]
 [Genome tool]
 
-## Analytics
+## Data Pipeline
+
+We use a set of [Rake](https://github.com/ruby/rake) tasks run on [Jenkins](https://wiki.jenkins-ci.org/display/JENKINS/Build+Flow+Plugin) to move data from several MongoDB and PostgreSQL databases to [AWS RedShift](https://aws.amazon.com/redshift) via [S3](https://aws.amazon.com/s3/). We also store application usage data provided by [Segment Warehouses](https://segment.com/warehouses) as well as data from vendors such as [Salesforce](https://www.salesforce.com/) and [Sailthru](http://www.sailthru.com/).
```

Comment:
Maybe say a little bit about what those rake tasks actually do? Like is it custom code? Some system? And why?